CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - `otelmux`: Add new `WithSpanNameFormatter` option to `go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux` to allow customizing span names. (#3041)
 - Improve documentation for `samplers/jaegerremote` by providing examples of sampling endpoints. (#3147)
 
+### Changed
```

Comment:
should this be Removed instead of changed?  Or otherwise marked breaking?

Reply:
I'd be fine with moving to `Removed` but I don't think this counts as breaking as it relates to attributes emitted by telemetry and not package APIs.